parcels/kernels/__init__.py

```diff
@@ -0,0 +1 @@
+from .advection import *  # noqa
```

Comment:
Since we want users to add kernels to this  `kernels` directory, would it make more sense to have it directly in the root of the repo? I.e. not a subdirectory of `parcels`?